content/en/developers/metrics/types.md

```diff
@@ -2,298 +2,327 @@
 
-## Metric type definition
+- [Custom Agent check][4]
+- [DogStatsD][5]
+- [Datadog’s HTTP API][6]
 
-To better understand the different metrics types, what they represent, and how to modify them within Datadog, consider the following example:
+The majority of data Datadog receives is submitted via the Agent (either Agent check or DogStatsD). For these submission methods, a metric’s type determines how multiple values collected on an Agent in [a flush time interval of 10s][7] are aggregated. The Agent combines these values into a single representative metric value for that interval. This combined value is stored with a single timestamp in Datadog.
 
-You have two web servers: `server:web_1` and `server:web_2`. Both web servers continuously receive:
+Data submitted directly to the Datadog API is not aggregated by Datadog (with the exception of distribution metrics) -- the raw values sent to Datadog are stored as-is.
 
-* 10 HTTP requests for the first 30 seconds, then
-* 20 HTTP requests for the next 30 seconds, then
-* 0 HTTP requests for the next 30 seconds.
+Refer to the Submission and In-App Types section to see how different metric submission types are mapped to their corresponding in-app types.
 
-### Metric submission types
+## Metric types
+
+### Definition
 
 {{< tabs >}}
 {{% tab "COUNT" %}}
 
-**The `COUNT` metric submission type represents the number of events that occur in a defined time interval, otherwise known as a flush interval**. This number of events can accumulate or decrease over time -- it is not monotonically increasing. A `COUNT` can be used to track the number of requests hitting your webservers or number of errors.
+The COUNT metric submission type represents the total number of event occurrences in one time interval. A COUNT can be used to track the total number of connections made to a database or the total number of requests to an endpoint. This number of events can accumulate or decrease over time – it is not monotonically increasing.
 
-**Note**: A `COUNT` is different from the `RATE` metric type, which represents this count of events normalized _per second_ given the defined time interval.
+**Note**: A COUNT is different from the RATE metric type, which represents this count of events normalized per second given the defined time interval.
 
-For example, suppose the `number.of.requests.count` metric is reported every 30 seconds to Datadog with the `COUNT` type on `server:web_1`.
+{{% /tab %}}
+{{% tab "RATE" %}}
 
-Each value/data point for this metric submitted as a `COUNT` represents the number of requests—the "count" of requests—received during the 30 second flush interval. The metric then reports the following values:
+The RATE metric submission type represents the total number of events occurrences per second. A RATE can be used to track how often something is happening—like the frequency of connections made to a database or the flow of requests made to an endpoint.
```

Comment:
The RATE metric submission type represents the total number of events occurrences per second in one time interval. A RATE can be used to track how often something is happening—like the frequency of connections made to a database or the flow of requests made to an endpoint.